src/network/network_content.cpp

```diff
@@ -474,8 +462,8 @@ bool ClientNetworkContentSocketHandler::Receive_SERVER_CONTENT(Packet &p)
 		}
 	} else {
 		/* We have a file opened, thus are downloading internal content */
-		size_t to_read = p.RemainingBytesToTransfer();
-		if (to_read != 0 && static_cast<size_t>(p.TransferOut(TransferOutFWrite, std::ref(this->cur_file))) != to_read) {
+		ssize_t to_read = p.RemainingBytesToTransfer();
+		if (to_read != 0 && p.TransferOut([this](std::span<const uint8_t> buffer) { return fwrite(buffer.data(), 1, buffer.size(), *this->cur_file); }) != to_read) {
```

Comment:
I think the dominant formatting (and the one I choosed for clang-format) is:
```suggestion
		if (to_read != 0 && p.TransferOut([this](std::span<const uint8_t> buffer) {
				return fwrite(buffer.data(), 1, buffer.size(), *this->cur_file);
			 }) != to_read) {
```

Reply:
I dislike it when the end of the if-statement is on the same indent as the 'scope' of the 'true' branch. But there are other solutions.